### ISSUE TITLE
Naming/RescuedExceptionsVariableName causes undefined method `loc' for nil:NilClass

### ISSUE THREAD

**Issue by znz** [rubocop-hq/rubocop#7167]
## Expected behavior

Do not raise error.

## Actual behavior

- `rescue` without variable name after `rescue` with a variable name
- auto correct

raise error

> undefined method `loc' for nil:NilClass

## Steps to reproduce the problem

```
% cat a.rb
def foo
rescue => ex
end

def bar
rescue
end
% rubocop -a a.rb
Inspecting 1 file


0 files inspected, no offenses detected
undefined method `loc' for nil:NilClass
.../rubocop-0.71.0/lib/rubocop/cop/naming/rescued_exceptions_variable_name.rb:102:in `location'
```

## RuboCop version

> 0.71.0 (using Parser 2.6.3.0, running on ruby 2.6.3 x86_64-darwin18)

**Comment by koic** [comment]
I could reproduce it and opened a PR #7168. Thanks!

**Comment by koic** [comment]
This issue has been resolved by #7122.
